scripts/claude-slide/generate_slide.py

```
@@ -169,13 +169,22 @@ def _embed(m: "re.Match[str]") -> str:
 
 
 def extract_html(response: str) -> str:
-    match = re.search(r"```html\s*\n(.*?)```", response, re.DOTALL)
+    # 非貪欲 (.*?) だと、生成 HTML の中に ``` が現れた時点で切れる（CSS コメントや
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)